cmd/dbbench/dbbench.go

```diff
@@ -115,16 +118,156 @@ type (
 		ranges         []IORange
 		totalFrequency int
 	}
+	// KeyValueDB directly exposes the necessary methods of leveldb.DB that we need to run the test so that they can be
+	// implemented by other KV stores
+	KeyValueDB interface {
+		Close() error
+		Compact() error
+		NewIterator() iterator.Iterator
+		Get([]byte) ([]byte, error)
+		Put([]byte, []byte) error
+	}
+	LevelDBWrapper struct {
+		ro     *opt.ReadOptions
+		wo     *opt.WriteOptions
+		handle *leveldb.DB
+	}
+	PebbleDBWrapper struct {
```

Comment:
nit: I would move most of the logic related to PebbleDB in `pebbledb.go` and same thing for LevelDB in `leveldb.go` just to keep it clean.

Reply:
I can move the pebbledb and leveldb implementations. I think that the common interferface between the two makes sense in the main package with the command.  RandomKeySeeker, I have no point of view, it's just a utility class that I needed for now I'll probably leave it where it is